lib/tool-bar-button-view.coffee

```diff
@@ -34,3 +37,14 @@ module.exports = class ToolBarButtonView extends View
 
   destroy: ->
     @subscriptions.dispose()
+
+  getPreviouslyFocusedElement: () ->
+    if @previouslyFocusedElement[0] and @previouslyFocusedElement[0] isnt document.body
+      @eventElement = @previouslyFocusedElement[0]
+    else
+      @eventElement = atom.views.getView(atom.workspace)
+
+  storeFocusedElement: ->
+    activeElement = $(document.activeElement)
+    if !activeElement.hasClass 'tool-bar-btn'
```

Comment:
If you change this line to `if not activeElement.classList.contains 'tool-bar-btn'` you can drop jQuery. 

Was thinking of refactoring the whole package to drop jQuery. Seen some core packages that drop jQuery in favor of vanilla JavaScript.